cylc/flow/scripts/validate.py

```diff
@@ -37,20 +38,44 @@
     TriggerExpressionError
 )
 import cylc.flow.flags
-from cylc.flow.id_cli import parse_id
+from cylc.flow.id_cli import parse_id_async
 from cylc.flow.loggingutil import disable_timestamps
 from cylc.flow.option_parsers import (
     WORKFLOW_ID_OR_PATH_ARG_DOC,
     CylcOptionParser as COP,
     Options,
     icp_option,
 )
+from cylc.flow.pathutil import (
+    get_source_conf_from_id, is_relative_to, get_cylc_run_dir)
 from cylc.flow.profiler import Profiler
 from cylc.flow.task_proxy import TaskProxy
 from cylc.flow.templatevars import get_template_vars
 from cylc.flow.terminal import cli_function
 
 
+def _revalidate(
+    cfg, flow_file, options, workflow_id, mem_log_func=None
+):
+    # Check if this is a rundir - revalidate not allowed otherwise:
```

Comment:
Should rename "revalidate" to something more informative e.g. "validate_against_installation"